x-pack/plugins/ml/public/application/explorer/alerts/alerts_summary.tsx

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiBadge, EuiDescriptionList, EuiFlexGroup, EuiFlexItem, EuiTitle } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { ALERT_DURATION, ALERT_END } from '@kbn/rule-data-utils';
+import React, { useMemo } from 'react';
+import useObservable from 'react-use/lib/useObservable';
+import { getAlertFormatters } from '../../../alerting/anomaly_detection_alerts_table/render_cell_value';
+import { useMlKibana } from '../../contexts/kibana';
+import { useAnomalyExplorerContext } from '../anomaly_explorer_context';
+import { getAlertsSummary } from './get_alerts_summary';
+
+export const AlertsSummary: React.FC = () => {
+  const {
+    services: { fieldFormats },
+  } = useMlKibana();
+  const { anomalyDetectionAlertsStateService } = useAnomalyExplorerContext();
+
+  const alertsData = useObservable(anomalyDetectionAlertsStateService.anomalyDetectionAlerts$, []);
+  const formatter = getAlertFormatters(fieldFormats);
+
+  const sortedAlertsByRule = useMemo(() => {
+    return getAlertsSummary(alertsData);
+  }, [alertsData]);
+
+  return (
+    <EuiFlexGroup>
+      {sortedAlertsByRule.map(([ruleName, ruleSummary]) => {
+        return (
+          <EuiFlexItem key={ruleName} grow={false}>
+            <EuiFlexGroup gutterSize="s" alignItems="center">
+              <EuiFlexItem grow={false}>
+                <EuiTitle size={'xs'}>
+                  <h5>{ruleName}</h5>
+                </EuiTitle>
+              </EuiFlexItem>
+              {ruleSummary.activeCount > 0 ? (
+                <EuiFlexItem grow={false}>
+                  <EuiBadge color="accent">Active</EuiBadge>
```

Comment:
Shouldn't this be internationalized?

Reply:
Done in https://github.com/elastic/kibana/commit/740fe6bdaf906d1ed078c8dc769cfb22bb513a5a